nilearn/glm/first_level/first_level.py

```diff
@@ -220,6 +220,38 @@ def run_glm(Y, X, noise_model='ar1', bins=100,
         results = {0.0: ols_result}
 
     return labels, results
+  
+  
+def _check_trial_type(events):
+  
+    """ Raises warning if the events.tsv file
+    doesn't contain column named "trial_type".
+      
+    parameters
+    ----------
+    events : :obj:`list` of path of events.tsv files
+
+    """
+    file_names = ''
+    
+    for event in events:
+        df = pd.read_csv(event, sep='\t', index_col=None)
+        if len(df.columns) > 2 and 'trial_type' not in df.columns:
+           file_names += f"{event.split('/')[-1]}, "
```

Comment:
I would rely on pathlib for this:

```python
from pathlib import Path

file_names = []
for event in events:
    event = Path(event)
    df = pd.read_csv(...)
    if ....:
        file_names.append(event.name)
if file_names:
    warn(f".... {','.join(file_names)}....")
```

Reply:
Also maybe easier to go for name the variable `event_` rather than just `event` to a make it easier to tell it apart from `events`.